service/static/js/index.js

```diff
@@ -2,12 +2,14 @@ $(function () {
   // ****************************************
   //  U T I L I T Y   F U N C T I O N S
   // ****************************************
+  const STRING_LIMIT = 64499;
```

Comment:
@yx2512 is this math correct? I always thought the length in the column is 1:1 to string length.

Reply:
Except for that, the column size you defined is 64. I think in PostgreSQL, it means 64 characters instead of 64499 characters.